pkg/minikube/cni/disabled.go

```diff
@@ -47,5 +47,6 @@ func (c Disabled) Apply(r Runner) error {
 
 // CIDR returns the default CIDR used by this CNI
 func (c Disabled) CIDR() string {
-	return ""
+	// Even without any CNI we want our nodes to have spec.PodCIDR set.
+	return DefaultPodCIDR
```

Comment:
why disabled need to return DefaultPodCIDR since right now we already change default cni as kindnet for both single node and multi nodes, and kindnet already has spec.PodCIDR.

func (c KindNet) CIDR() string {
	return DefaultPodCIDR
}

Reply:
Actually it doesn't now as we set kindnet as default. But now that we are enabling `allocate-node-cidrs` the podCIDR value can't be empty. Empty podCIDR will result in kube-controller-manager errors. Keeping this as failsafe/default value if someone force uses disabled cni via https://github.com/kubernetes/minikube/blob/master/pkg/minikube/cni/cni.go#L79.

Reply:
yes.